README.md

```diff
@@ -170,7 +175,7 @@ Create a data frame from a file named `sg-weather.csv` on the local disk:
 
 ```lisp
 (defparameter *df*
-	(read-csv #P"LS:DATASETS;sg-weather.csv"))
+  (read-csv #P"LS:DATASETS;sg-weather.csv"))
```

Comment:
I just noted a problem here. It should now be #P"LS:DATA;sg-weather.csv". Could you please update this line as well?